app/game/components/game-input/game-input.js

```diff
@@ -10,10 +10,14 @@ import { verifySolution } from '../../action-creators/round';
 import './game-input.styl';
 
 class GameInput extends Component {
-    render({ result, dispatch, currentRoundIndex }) {
+    render({ result, dispatch, playerInput, currentRoundIndex }) {
         return (
             <div className="game-input">
-                <CodeEditor onChange={bindActionCreators(verifySolution, dispatch)} currentRoundIndex={currentRoundIndex} />
+                <CodeEditor
+                    onChange={bindActionCreators(verifySolution, dispatch)}
```

Comment:
True, but there's also written that `bindActionCreators()` can be used to bind actions. Don't really matter for me. Fixed.